wpilibc/src/main/native/include/frc/buttons/POVButton.h

```diff
@@ -24,6 +24,6 @@ class POVButton : public Button {
  private:
   int m_angle;
   int m_povNumber;
-  GenericHID &m_joystick;
+  GenericHID& m_joystick;
```

Comment:
To make the compiler automatically generate a move constructor for you when you declare it as default, you can make this a pointer and do `m_joystick(&joystick)` in the constructor initializer list. (References can only be assigned to once, so the default would be ill-formed in that case.)